client/src/elements/organisms/tool-status-bar.html

```diff
@@ -7,9 +7,8 @@
   </div>
 
   <div class="tool-status-bar__group">
-    <icon-active-state is-active.bind="item.conf.active" class="tool-status-bar__item" ></icon-active-state>
+    <icon-active-state is-active.bind="item.conf.active" class="tool-status-bar__item"></icon-active-state>
     <button-secondary if.bind="item.conf.active" class="tool-status-bar__item" icon="deactivate" click.delegate="onDeactivateClick()"></button-secondary>
```

Comment:
as there is enough space, we should add the activate/deactivate text to this button.
same for the save button.

Reply:
Text is now shown next to the icon if there is enough space available (won't be show on small screen sizes) -> https://github.com/nzzdev/Q-editor/pull/116/commits/81d6d13d2feffb337495807de9be6deb35d1c331